docs/getting-started/what-is-dotnet.md

```diff
@@ -95,15 +95,21 @@ Other libraries are installed as [NuGet](http://www.nuget.org/ "NuGet") packages
 
 ## .NET Core CLI ##
 
-.NET Core includes a new cross-platform command-line SDK known as the CLI (for Command-Line Interface). The CLI is a UNIX-friendly set of tools for building .NET Core applications. It abstracts the C# compiler, the NuGet package manager, and includes incremental build logic. It can also integrate with the .NET Native tool chain to produce high-performance native apps and libraries. The CLI allows developers to build and test applications with a minimal installation on developer, lab, or production machines. Since some IDEs, such as Visual Studio Code nad Visual Studio, use the CLI under the covers, developers can now choose the level they want to interact with the tool chain by working with the CLI directly, using an editor that uses the CLI, or developing with an IDE that uses the CLI internally.  
+.NET Core includes a new cross-platform command-line SDK known as the CLI (for Command-Line Interface). The CLI is a UNIX-friendly set of tools for building .NET Core applications. It abstracts the C# compiler, the NuGet package manager, and includes incremental build logic. It can also integrate with the .NET Native tool chain to produce high-performance native apps and libraries. The CLI allows developers to build and test applications with a minimal installation on developer, lab, or production machines. Since some IDEs, such as Visual Studio Code and Visual Studio, use the CLI under the covers, developers can now choose the level they want to interact with the tool chain by working with the CLI directly, using an editor that uses the CLI, or developing with an IDE that uses the CLI internally.  
```

Comment:
>Since some IDEs, such as Visual Studio Code and Visual Studio, use the CLI under the covers, developers can now choose the level they want to interact with the tool chain by working with the CLI directly, using an editor that uses the CLI, or developing with an IDE that uses the CLI internally.  

What does that mean? What's the important info from that? Currently way too long of a run-on sentence.
VS Code and VS use CLI, therefore you can choose the level of interactions with the tool chain. For example, you can let the IDE .... or you can use the command line (CLI) - or something like that.